src/app/portal/features/account/account.component.html

```diff
@@ -0,0 +1,81 @@
+<h1 class="text-center">
+  <span class="text-3xl">User Account</span><br />
+  <span class="text-3xl">{{ auth.fullName() }}</span>
+</h1>
+
```

Comment:
This should also show whether you're logged in as a data steward somewhere. (In the legacy app this was shown in the green message at the top.)